.changeset/purple-kiwis-sit.md

```diff
@@ -0,0 +1,7 @@
+---
+'@commercetools-docs/gatsby-theme-docs': major
```

Comment:
PS: why major?  I can't see a breaking change here.

Reply:
Agree, it should be minor

```suggestion
'@commercetools-docs/gatsby-theme-docs': minor
```